rosdistro_reviewer/submitter/github.py

```diff
@@ -16,9 +16,30 @@
+This is an automated tool that helps check your pull request for correctness. This tool checks a number of attributes associated with your ROS package and generates a report that helps our reviewers merge your pull request in a timely fashion. Here are a few things to consider when sending adding or updating a package to ROS Distro. ROS Distro includes a very helpful [CONTRIBUTING.md](https://github.com/ros/rosdistro/blob/master/CONTRIBUTING.md) file that we recommend reading if it is your first time submitting a package. Please also read the [ROS Distro review guidelines](https://github.com/rosdistro/rosdistro/blob/master/REVIEW_GUIDELINES.md) which summarizes this release process.
+
+If you'd like to run this tool again to generate a new review for your PR, just [re-request review from GitHub Actions](https://github.com/ros-infrastructure/rosdistro-reviewer?tab=readme-ov-file#running-rosdistro-reviewer-in-github-actions).
+
+# ROS Distro Considerations
+* ROS Distribtutions are created using [REP-134 Standards Track](https://ros.org/reps/rep-0143.html) as a guide.
+* Your package name should comply to [REP-144 ROS Package Naming](https://www.ros.org/reps/rep-0144.html)
+* Your pacakge must build for all platforms and architectures on the ROS buildfarm. See [REP-2000 ROS Releases and Supported Platforms](https://www.ros.org/reps/rep-2000.html) for all supported platforms for your ROS Distro.
+* Your package must contain an [OSI approved license](https://opensource.org/licenses). Your `package.xml` file must also include that license in a machine readable format. See [REP-149 Package Manifest Format Three Specification](https://ros.org/reps/rep-0149.html#license-multiple-but-at-least-one) for additional details.
+* Your package must include a copyright at the top of every file.
+* A publicly available, open source, repository for your ROS package.
+* While not required, we recommend that you create an account for ROS Discourse and subscribe to the [appropriate release topic](https://discourse.ros.org/c/release/16).
+* If you real time chat would help you resolve an issue please join our [Discord Server] and join the `#infra-help` channel.
+
+# Package Considerations
+
+Having your package included in a ROS Distro is a badge of quality, and we recommend that package developers strive to create packages of the highest quality. We recommend package developers review the following resources before submitting their package.
+
+* [REP-2004 Package Quality Declaration](https://www.ros.org/reps/rep-2004.html)-- The ROS 2 TGC has created a quality rating system for ROS packages. These ratings should serve as a guide for package developers. We recommend package developers achieve a quality level of three or higher.
```

Comment:
```suggestion
* [REP-2004 Package Quality Declaration](https://www.ros.org/reps/rep-2004.html)-- The ROS 2 TSC has created a quality rating system for ROS packages. These ratings should serve as a guide for package developers. We recommend package developers achieve a quality level of three or higher.
```

The Technical Steering Committee was the governing body when the quality levels were established although I don't recall what level of involvement they had in actually drafting and ratifying these. I recommend referring to these as ROS standards since they went through the REP process but I'm soft on it.